sql/log_event.cc

```diff
@@ -10500,6 +10500,30 @@ int Rows_log_event::handle_idempotent_and_ignored_errors(Relay_log_info const *r
                            (rbr_exec_mode == RBR_EXEC_MODE_IDEMPOTENT));
     bool ignored_error= (idempotent_error == 0 ?
                          ignored_error_code(actual_error) : 0);
+#if defined(WITH_WSREP)
+    /*
+     * Work around mysql Bug#80821. With cascading deletes, when both parent and
+     * child tables are involved in a multi table query, a delete rows event for
+     * the child may be added to the binlog after the parent (cascading) delete.
+     * This will cause the delete to fail on slaves. A new wsrep_mode value has
+     * been added for ignoring such delete errors.
+     */
```

Comment:
IIUC, [Bug#80821](https://bugs.mysql.com/bug.php?id=80821) is already fixed in PS and PXC. So I'm not sure if this is really necessary.

@dutow Can you please share your opinions on this?